ros_buildfarm/templates/status/js/setup.js

```diff
@@ -218,8 +218,16 @@ function scan_rows() {
   $('table tbody tr').each(function() {
     // Add lowercased version of name (which is the last meta column) for faster case-insensitive search.
     var name_td = $('td:nth-child(' + window.META_COLUMNS + ')', this);
+    var hidden_text = '';
     if (name_td.text() && name_td.text().length > 0) {
-      name_td.append(' <span class="ht">' + name_td.text().toLowerCase() + '</span>');
+      hidden_text += name_td.text().toLowerCase()
+    }
+    // Add a copy of the email to the hidden text for additional searching
+    name_td.find('a').each(function() {
+      hidden_text += this['href'];
```

Comment:
Should the `mailto:` prefix be stripped from this?

Reply:
The text isn't visually seen. The existing code doesn't even insert spaces between names. However, if you're requesting that it strip mailto, I can do that.

Reply:
> The text isn't visually seen.
> However, if you're requesting that it strip mailto, I can do that.

Sure, but then you could search for "mailto" which doesn't seem to be desired. Please remove the prefix.

> The existing code doesn't even insert spaces between names.

Maybe adding at least a space or even a separator would be good to avoid weird search results.